src/resources/ble_darwin.mm

```diff
@@ -1207,6 +1207,10 @@ _new BleRemoteDeviceResource(
   UNIMPLEMENTED();
 }
 
+PRIMITIVE(handle) {
+  UNIMPLEMENTED();
```

Comment:
Maybe these primitives should be using `FAIL(UNIMPLEMENTED)`, so we don't crash the VM for no good reason?

Reply:
I will do this in a follow-up PR.